WebHostLib/static/assets/gameInfo/en_VVVVVV.md

```diff
@@ -0,0 +1,35 @@
+# VVVVVV
+
+## Where is the settings page?
+The player settings page for this game is located <a href="../player-settings">here</a>. It contains all the options
```

Comment:
You can use a markdown native link for this, just use the relative path from root URI to the player-settings page.

Reply:
I've copied this part from another doc page; it seems that all of them use it. Should I still change it?

Reply:
Yes. I know it's been copied, and I believe I've changed them all in my docs branch.

Reply:
Oh, didn't know there was work being done on the docs. Amended